images/win/scripts/Installers/Install-Apache.ps1

```diff
@@ -0,0 +1,28 @@
+################################################################################
+##  File:  Install-Apache.ps1
+##  Desc:  Install Apache HTTP Server
+################################################################################
+
+# Stop w3svc service
+Stop-Service -Name w3svc 2>&1 | Out-Null
+
+# Install latest apache in chocolatey
+$installDir = "c:\tools"
+Choco-Install -PackageName apache-httpd -ArgumentList "--force", "--params", "/installLocation:$installDir /port:80"
+
+# Set the APACHEROOT environment variable
+$apacheDirName = (Get-ChildItem $installDir\apache*).Name
+setx APACHEROOT $installDir\$apacheDirName /M
+
+# Add Apache bin directory to PATH
+Add-MachinePathItem $installDir\$apacheDirName\bin
```

Comment:
Please don't add anything to PATH, it contains a lot of stuff.

Reply:
Removed